STYLE.md

```diff
@@ -131,15 +131,188 @@ if status is not pebble.ServiceStatus.ACTIVE:
 
 ## Docs and docstrings
 
-### Use British English
+### Tones
+
+#### Avoid negative, be positive
+
+State conditions positively as what should happen, rather than what shouldn't. Focus on desired behaviour, and frame instructions and conditions around the expected or successful outcome, not failure cases.
+
+Example:
+
+- Avoid: "If the command doesn’t exit within 1 second, the start is considered successful." (Netagive)
+- Prefer: "If the command stays running for the 1-second window, the start is considered successful." (Positive)
+
+Exceptions: Only use negative phrasing (e.g., "if X does not happen") when:
+
+- The failure case is the primary concern (e.g., error handling).
+- The positive phrasing is awkward or less clear.
+
+#### Avoid passive, be active
+
+Use the active voice as much as possible.
+
+We should only use the passive voice when we don't know or care about who performed the action.
+
+Example:
+
+- Avoid: "A minimum check is created using the default values"
+- Prefer: "We create a minimum check, using the default values"
+
+#### Avoid subjective, be objective
+
+Instead of using words like "easy," "simple," or "just," which are very subjective and assume the reader's skill level, describe the action directly by stating what to do and focusing on concrete steps.
+
+Examples:
+
+- Avoid: "This can be done easily using ..."
+- Prefer: "This can be done using ..."
+
+- Avoid: "This can be easily configured by ..."
+- Prefer: "We can configure this using ..."
+
+- Avoid: "Simply run the command ..."
+- Prefer: "Run the command ..."
+
+### Structure
+
+#### Be short
+
+For example, a very long introduction part of a how-to guide or a tutorial probably isn't necessary; it's good if we are writing a blog post or a book, but not so much for a tutorial/how-to guide type of doc.
+
+#### Be concise and reduce repetition
+
+Trim repetitive titles: Avoid repeating the name if the surrounding content already provides context.
```

Comment:
What is "the name"? Maybe join the following "For example" paragraph with this one.